react/features/filmstrip/actionTypes.js

```diff
@@ -118,3 +118,44 @@ export const SET_USER_FILMSTRIP_WIDTH = 'SET_USER_FILMSTRIP_WIDTH';
  * }
  */
 export const SET_USER_IS_RESIZING = 'SET_USER_IS_RESIZING';
+
+/**
+ * The type of (redux) action which sets the dimensions of the thumbnails in stage filmstrip view.
+ *
+ * {
+ *     type: SET_STAGE_FILMSTRIP_DIMENSIONS,
+ *     dimensions: Object
+ * }
+ */
+export const SET_STAGE_FILMSTRIP_DIMENSIONS = 'SET_STAGE_FILMSTRIP_DIMENSIONS';
+
+/**
+ * The type of Redux action which adds a participant to the active list
+ * (the participants displayed on the stage filmstrip).
+ * {
+ *     type: ADD_ACTIVE_PARTICIPANT,
+ *     participantId: string,
+ *     pinned: boolean
+ * }
+ */
+export const ADD_ACTIVE_PARTICIPANT = 'ADD_ACTIVE_PARTICIPANT';
```

Comment:
name is a bit misleading imo, maybe something like `ADD_STAGE_PARTICIPANT` along with the name changes for derived functionality..wdyt?